requirements.txt

```
@@ -5,11 +5,12 @@ cfn_lint>=0.72.0,<1.0
 setuptools>=40.4.3
 boto3>=1.9.21,<2.0
 botocore>=1.12.21,<2.0
+urllib3<2
 yattag>=1.10.0,<2.0
-PyYAML~=5.1
+PyYAML~=6.0.1
```

`~=` better use two number of semver

```suggestion
PyYAML~=6.0
```